modules/ROOT/pages/6.4-release-notes.adoc

```diff
@@ -0,0 +1,1177 @@
+
+As a result, in {productname} 6.4, when using the backspace or delete function keys on `contenteditable="false"` elements within a `contenteditable="false"` element, the contents within the `contenteditable="false"` elements will no longer be deleted. 
+
+=== Inserting newlines inside an editable element inside a noneditable root would sometimes try to split the editable element
+// TINY-9461
+
+In previous versions of {productname}, inserting newlines inside an `contentEditable= true;` element while it was inside a `contentEditable= false;` root would sometimes try to split the `contentEditable= true;` element. 
+
+As a consequence, when child block elements set to `contentEditable=true;` such as a `<p>` tag were nested within the `contentEditable= false;` element root, the editor would try to split the child block element when the user hit the `enter` command. 
+
+As the paragraph tag is intended to be a block for `text`, it is classified as `semantically incorrect` to split a paragraph block element.
+In this particular instance, child elements in {productname} such as the `<p>` tag would split the block element resulting in a new block (such as a new paragraph) to be created inside the paragraph when a user attempted to edit it inside a `contentEditable= false;` root element. 
+
+Fixes were introduced with the release of {productname} 6.4, that:
+
+* Prevents editing host elements like `<p contenteditable="true">paragraph text</p>` in a `contentEditable= false;` root to be split.
+* Prevents editing hosts elements inside a `contentEditable= false;` root to be split.
+
+As a result, when a root element is set to `contentEditable= false;`, any `contentEditable= true;` elements inside that root element such as `<p>` tag will no longer try to split the `contentEditable= true;` element. 
+
+=== Creating a list in a table cell when the caret is in front of an anchor element would not properly include the anchor in the list.
+// TINY-6853
+
+In previous versions of {productname}, a bug was causing issues with `links` while they were nested within table-cells. As a consequence, it was found that when creating a `num-list` or `bullet-list` to the left (node) of a `link` that was nested within a table cell the `list` would *not* include the created `link` within the `num or bullet-list` as expected. 
+
+To fix this bug, the table cell now checks *If* the caret position is in front of an *anchor element*, `if` (true), then the table cell will include the anchor in the `list`.
+
+In {productname} 6.4, anchor elements are included inside `lists` while nested within table cells.
+
+=== Dragging and dropping noneditable elements on table borders would remove the element on drop
+// TINY-9021
+
+In previous versions of {productname}, dragging and dropping `contentEditable= false;` elements on table borders would create issues during the drop action.
```

Comment:
@kemister85 , there are a lot of this pattern `contentEditable= false;`  or `contentEditable= true;` in the doc. Please correct them to `contenteditable="false"` or `contenteditable="true"` respectively

Reply:
@tiny-ben-tran these have all been updated, thanks for the pickup.